core/src/main/java/hudson/security/Permission.java

```diff
@@ -315,7 +306,26 @@ public boolean getEnabled() {
      *      Access {@link jenkins.model.Jenkins#ADMINISTER} instead.
      */
     @Deprecated
-    public static final Permission HUDSON_ADMINISTER = new Permission(HUDSON_PERMISSIONS,"Administer", hudson.model.Messages._Hudson_AdministerPermission_Description(),null);
+    public static final Permission HUDSON_ADMINISTER;
+
+    @Restricted(NoExternalUse.class)
+    public static final Permission ROOT;
+
+    static {
+        final Permission administer, runScripts, root;
```

Comment:
Could assign directly to the actual constants I think.

Reply:
True. I'll clean this up.